update doc to match config default value with TiKV codebase. (#5188)
pingcap/docs: tikv-configuration-file.md

### `grpc-memory-pool-quota`

+ Limit the memory size that can be used by gRPC

> + Default: unlimited

Reviewer: ```suggestion
+ Default: No limit
```

### `use-unified-pool`

+ Determines whether to use the unified thread pool (configured in [`readpool.unified`](#readpoolunified)) for storage requests. If the value of this parameter is `false`, a separate thread pool is used, which is configured through the rest parameters in this section (`readpool.storage`).

> + Default value: If this section (`readpool.storage`) has no other configurations, the default value is `true`. Otherwise, to upgrade compatibility, the default value is `false`. Change the configuration in [`readpool.unified`](#readpoolunified) as needed before enabling this option.

Reviewer: ```suggestion
+ Default value: If this section (`readpool.storage`) has no other configurations, the default value is `true`. Otherwise, for the forward compatibility, the default value is `false`. Change the configuration in [`readpool.unified`](#readpoolunified) as needed before enabling this option.
```

> + Default value: If this section (`readpool.storage`) has no other configurations, the default value is `true`. Otherwise, for the forward compatibility, the default value is `false`. Change the configuration in [`readpool.unified`](#readpoolunified) as needed before enabling this option.

Reviewer 2: I think it's for backward compatibility.

### `stats-dump-period`

> + The interval time that outputs statistics to the log.

Reviewer: ```suggestion
+ The interval at which statistics are output to the log.
```
